hindi/joke/joke.md

```diff
@@ -1,3 +1,4 @@
+[here](https://github.com/udacity/create-your-own-adventure/blob/master/hindi/Akshay_Kumar/akshay_kumar.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.